generator/generator_md.py

```diff
@@ -0,0 +1,96 @@
+import sys
```

Comment:
Your pull request GENERATOR-1022 Added generator_py.md/n

Your commit: GENERATOR-1022 Added generator_md.py

Your commit: GENERATOR-1022 Added other generator files

Your commit: GENERATOR-1022 Added testgenerator.py

Your commit: GENERATOR-1022 Refactored generator_md.py

Your commit: GENERATOR-1022Added testgenerator.py
The group number of your message must contain group number in ['1011', '1012', '1021', '1022']/nThe action of your message must contain action in ['Added', 'Deleted', 'Refactored', 'Fixed', 'Moved']